pkg/crdupdater/replicationcontroller.go

```diff
@@ -141,10 +140,10 @@ func (r *ReplicationController) remove() error {
 // replicationControllerComparator used to compare Replication controller attributes
 type replicationControllerComparator struct {
 	Image          string
-	MinCPU         *resource.Quantity
-	MaxCPU         *resource.Quantity
-	MinMem         *resource.Quantity
-	MaxMem         *resource.Quantity
+	MinCPU         string
```

Comment:
Is a `replicationControllerComparator` needed at all anymore, once we move everything to `Deployment`?